nifpga/fixedpointhelper.py

```diff
@@ -0,0 +1,59 @@
+from warnings import warn
+
+
+def bin_to_int(binary_string):
+    if binary_string == '':
+        return 0
+    else:
+        return int(binary_string, 2)
+
+
+def to_bin(value):
+    """
+    This class uses the built in bin() function to do the conversion,
```

Comment:
What class is "This class"?

Reply:
Removed in "Removing string based implementation commit"